content/en/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -1118,11 +1048,18 @@ Insecure values:
 <td></td><td style="line-height: 130%; word-wrap: break-word;">Topology Manager policy to use. Possible values: <code>'none'</code>, <code>'best-effort'</code>, <code>'restricted'</code>, <code>'single-numa-node'</code>. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)</td>
 </tr>
 
+<tr>
+<td colspan="2">--topology-manager-policy-options string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">A set of key=value Topology Manager policy options to use, to fine tune their behaviour. If not supplied, keep the default behaviour. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)</td>
```

Comment:
Are we trying to say that with this string we're specifying a key-value pair used by the Topology Manager along with the policy options to finetune its behavior and if they aren't specified, it resorts to default behavior?

Reply:
This is directly copy-pasted from the `kubelet` help message.
I agree to your understandings.